source/css/aboutus.css

```diff
@@ -0,0 +1,198 @@
+.card-deck.about {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding: 0;
+    margin: 0;
+}
+
+.card.about {
+    border: none;
+    border-radius: 0;
+    margin: 0;
+    margin-bottom: 1.25rem;
+}
+
+.card.about:last-child {
+    margin-bottom: 0;
+}
+
+div.card-img img {
+    border-radius: 0;
+}
+
+div.card-content {
+    background-color: #a60000;
+    color: white;
+}
+
+.card.about div.card-content div.card-body {
+    background-color: #dddddd;
+    color: black;
+    padding: 0;
+}
+
+.card.about div.card-content div.card-title {
+    margin: 0;
+}
+
+div.card-content div.card-title h3 {
+    margin: 0;
+    padding: 2%;
+    font-size: x-large;
+    font-family: 'Century Gothic', 'Futura', 'Didact Gothic', sans-serif;
+}
+
+div.card-text.short p {
+    display: -webkit-box;
+    overflow: hidden;
+    line-clamp: 3;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
+    text-overflow: ellipsis;
+    -o-text-overflow: ellipsis;
+    -ms-text-overflow: ellipsis;
+    transition: .25s linear;
+}
+
+div.card-text.short p.full {
+    -webkit-line-clamp: 20;
+    transition: .25s linear;
+}
+
+div.card-text.short,
+div.card-text.long {
+    padding: .75rem 1.25rem;
+}
+
+div.card-body div.card-text.long {
+    display: none;
+}
+
+.card.about div.card-footer {
+    padding: 0;
+    margin: 0;
+    border-top: 0;
+}
+
+div.card-footer button {
+    font-family: 'Century Gothic', 'Futura', 'Didact Gothic', sans-serif;
+    width: 100%;
+    height: auto;
+    border: 0;
+    background-color: #960000;
+    color: white;
+    padding: .75rem;
+    border-top: 1px solid rgba(0,0,0,.125);
+}
+
+@media screen and (min-width: 670px) {
+    .card-deck.about {
+        width: 100%;
+        padding: 0 2%;
+    }
+
+    .card.about {
+        margin-bottom: 5%;
+    }
+
+    .card.about:nth-child(odd) {
+        flex-direction: row;
+    }
+
+    .card.about:nth-child(even) {
+        flex-direction: row-reverse;
+    }
+
+    .card.about .card-content {
+        width: calc(55%*0.99);
+        margin: 0 auto;
+        position: relative;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        /* height: max-content; */
```

Comment:
why is this here / commented out?

Reply:
Testing out what looks better whether `height: auto;` or `height: max-content;`, we'll see over time so I'll just leave it there for now.